mmv1/products/redis/Cluster.yaml

```diff
@@ -250,6 +250,56 @@ properties:
     description: |
       System assigned, unique identifier for the cluster.
     output: true
+  - name: 'automatedBackupConfig'
+    type: NestedObject
+    description: "The automated backup config for a instance."
+    properties:
+      - name: 'automatedBackupMode'
+        type: Enum
+        description: |
+          "Optional. The automated backup mode. If the mode is disabled, the other fields will be ignored."
+          1. `AUTOMATED_BACKUP_MODE_UNSPECIFIED`: Default value. Automated backup config is not specified.
+          2. `DISABLED`: Automated backup config disabled.
+          3. `ENABLED`: Automated backup config enabled.
+        enum_values:
+          - 'AUTOMATED_BACKUP_MODE_UNSPECIFIED'
+          - 'DISABLED'
+          - 'ENABLED'
+      - name: 'fixedFrequencySchedule'
+        type: NestedObject
+        description: "Optional. Trigger automated backups at a fixed frequency."
+        properties:
+          - name: 'startTime'
+            type: NestedObject
+            description: |
+              Required. The start time of every automated backup in UTC.
+              It must be set to the start of an hour. This field is required.
+            required: true
+            properties:
+              - name: 'hours'
+                type: Interger
```

Comment:
I am surprised we don't return an error in some way for this! Should be `Integer` here + below

fyi @c2thorn